### ISSUE TITLE
License file

### ISSUE THREAD

**Issue by tnyeanderson** [TChilderhose/pulse-sms-backend#1]
Hi!

Thanks so much for providing this! You responded to my [reddit post](https://www.reddit.com/r/selfhosted/comments/k7297f/pulsesms_anyone_have_the_latest_source_code/)

I would like to use this code in my fork and flesh it out. You've really provided a great base.

But before I continue, I need to ensure that I have the rights to do so. Can you please add a LICENSE file? It's your code so you can license it however you'd like, but I think a copyleft license would be great :)

Thanks again!

**Comment by TChilderhose** [comment]
Hey! 

I added a License file. Just added MIT so that there is basically no restrictions.

Got your reddit message too. Re: Linux, yeah it should work on there. It uses Kestrel which I believe works basically the same on Windows and Linux now. I also added a README with a bunch of things on there. I left it in a pretty okay state I think. 90% of the api stuff was done, was really just some of the random calls that I didn't trigger and getting Firebase fully working for multiple clients. It was sending the messages to the client, but was expecting different names for stuff, which kind of sucks. I think I got a majority of the stuff in the readme, but let me know if you get stuck with something.
